package/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -222,16 +232,28 @@ private void updateAll() {
     updateValue();
   }
 
-  /** Update limit based on props limit, max and min */
+  /** Update limit based on props limit, max and min
+   * Fallback to upper limit if invalid configuration provided
+  */
   private void updateLowerLimit() {
     double limit = Math.max(mRealLowerLimit, mMinValue);
-    mLowerLimit = (int) Math.round((limit - mMinValue) / (mMaxValue - mMinValue) * getTotalSteps());
+    int lowerLimit = (int) Math.round((limit - mMinValue) / (mMaxValue - mMinValue) * getTotalSteps());
+    if(lowerLimit > mUpperLimit) {
+      Log.w("Invalid configuration", "reverting lower limit to upper limit");
```

Comment:
We shouldn't display logs in very different format for the same type of error.
Let's unify all of them to the one proposed in the suggestions for iOS platform (of course considering whether it's for `upperLimit` or `lowerLimit`).